R/slide.R

```diff
@@ -309,15 +309,16 @@ epi_slide <- function(
     ...,
     .keep = TRUE
   ) %>%
-    bind_rows() %>%
-    filter(.real) %>%
-    select(-.real) %>%
+    {
+      vec_rbind(!!!.)
```

Comment:
This incantation looks so strange. Is it much faster than bind_rows? Or does it have other benefits?

Reply:
At some point it seemed to give a substantial performance benefit, but now it seems like it might be a wash.  `vec_rbind` seems a bit cleaner in that it determines the output ptype by looking at all the arguments and finding something compatible, rather than `dplyr_reconstruct`-ing with the first argument as a template.  I do wish that they had a `list_rbind` (oh hey, there's one in `purrr`! though might need to check overhead).

The incantation is the combination of these two:
- `x %>% { f(g(.)) }` --- call f(g(x)), not, f(x, g(x))
- `vec_rbind(!!! list_of_data_frames)` (`vec_rbind` expects things like `vec_rbind(df1, df2, df3)`, but supports splatting)

Reply:
Side note: `vec_cbind` also seems to try to find a compatible ptype it could cast all the entries to... which doesn't make any sense.  I forget exactly why I moved to it; I think it seemed like `bind_cols` had some overhead and/or output `epi_df`s when working with tibbles was faster, and I didn't want to make things look visually even messier with `[<-`, though operation-wise that'd probably be cleanest.

Reply:
Immediate issue with `[<-` was actually recycling of the group key.  I found another approach that looks a little bit cleaner than `vec_cbind()` or `bind_cols()` IMO.  Also I moved to using `list_rbind()` above.